contracts/gauges/LiquidityGaugeV5Collateral.vy

```diff
@@ -733,6 +749,28 @@ def set_killed(_is_killed: bool):
 
     self.is_killed = _is_killed
 
+@external
```

Comment:
Admin can set an address of whitelist. In practice should only ever have one whitelisted address, which is whatever the custody contract is in the lending pool.